registry/registry_test.go

```diff
@@ -663,7 +663,7 @@ func TestMirrorEndpointLookup(t *testing.T) {
 		}
 		return false
 	}
-	s := DefaultService{config: makeServiceConfig([]string{"my.mirror"}, nil)}
+	s := DefaultService{config: makeServiceConfig([]string{"https://my.mirror"}, nil)}
```

Comment:
since it fails in the test, when validateMirror.
Mirror address must have scheme.